### PR TITLE
fix empty list of input handling

### PR DESCRIPTION
**Pull request by titaiwangms**
The bug is caught in PyTorch CI tests

### CODE REVIEW

**Comment by justinchuby** [comment]
Is https://github.com/microsoft/onnx-script/issues/386 related?

**Comment by titaiwangms** [comment]
> Is #386 related?

No, this happens in fixed shape model

File: onnxscript/function_libs/torch_aten/graph_building.py [hunk 1]
```diff
@@ -411,8 +411,11 @@ def _add_torchscript_op_call(
         graph_inputs = []
         assert isinstance(unwrapped_inputs, Sequence)
         for input in unwrapped_inputs:
-            if isinstance(input, Sequence) and all(
-                isinstance(elem, torch.Value) for elem in input
+            # NOTE(titaiwang): input could be empty list
+            if (
+                input
+                and isinstance(input, Sequence)
```
Comment (justinchuby) [inline onnxscript/function_libs/torch_aten/graph_building.py:417]:
```suggestion
                isinstance(input, Sequence)
                and input
```

**Review by justinchuby — APPROVED** [review]
nit

**Pull request merged**